src/components/Jobs/JobSelector/JobSelector.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { components } from 'react-select'
+import AsyncSelect from 'react-select/async'
+import { ReactComponent as DropDownIcon } from '../../../assets/icons/ic-chevron-down.svg'
+import { JobSelectorType } from '../Types'
+import { appSelectorStyle, jobListOptions } from './JobSelector.utils'
+
+const DropdownIndicator = (props) => {
```

Comment:
can we use the existing DropdownIndicator and noOptionsMessage from appSelectorUtil?

Reply:
Even can we please reuse the same selector if posssible?

Reply:
Yes. Removed this & now using the AppSelector.